packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.scss

```diff
@@ -64,7 +64,7 @@
     label,
     &__field:focus ~ &__label {
         position: absolute;
-        bottom: 5.5rem;
+        top: -0.5rem;
```

Comment:
Position the label in relation to the top, given that the parent's height may vary